TYPO3.Neos/Documentation/HowTos/ExtendingPages.rst

```diff
@@ -31,7 +31,7 @@ in the below example:
 
 TypoScript (Sites/Vendor.Site/Resources/Private/TypoScripts/Library/Root.ts2) ::
 
-	prototype(TYPO3.Neos:Page) {
+	prototype(TYPO3.Neos.NodeTypes:Page) {
```

Comment:
this ain't no typo.. `TYPO3.Neos:Page` is what's used to render pages

see https://github.com/neos/neos-development-collection/blob/2.3/TYPO3.Neos/Resources/Private/TypoScript/Prototypes/Page.ts2